zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSetting.java

```diff
@@ -30,87 +37,69 @@
   private static final String SHARED_PROCESS = "shared_process";
   private String id;
   private String name;
-  private String group;
-  private String description;
+  private String refName; // always be null in case of InterpreterSettingRef
   private Properties properties;
-  private transient InterpreterGroupFactory interpreterGroupFactory;
 
-  // use 'interpreterGroup' as a field name to keep backward compatibility of
-  // conf/interpreter.json file format
-  private List<InterpreterInfo> interpreterGroup;
+  @SerializedName("interpreterGroup")
+  private List<InterpreterInfo> interpreterInfos;
   private transient Map<String, InterpreterGroup> interpreterGroupRef = new HashMap<>();
   private List<Dependency> dependencies;
   private InterpreterOption option;
+  private transient String path;
+
+  @Deprecated
+  private String group;
```

Comment:
There're two reason. The most important point is mapping `interpreterSettingRef` and `interpreterSetting`. If we keep group as the name of interpreter, `InterpreterSetting.group` points `InterpreterSetting.name`. I think it makes more difficult to understand mapping. Currently, `InterpreterSetting.refName` points `InterpreterSettingRef.name`. I thought it would be better. Second, in more minor reason, This is a bit confusion from `InterpreterGroup`. `InterpreterSetting.group` is not related `interpreterGroup` but because those two terms includes `group` in their name, it looks related. It, however, is minor issue.